frontend/src/component/project/Project/ProjectOverview.tsx

```diff
@@ -63,12 +78,18 @@ const InfiniteProjectOverview = () => {
                 <ProjectStats stats={project.stats} />
                 <StyledProjectToggles>
                     <ProjectFeatureToggles
-                        key={loading ? 'loading' : 'ready'}
-                        features={searchFeatures.features}
+                        key={
+                            loading && dataList.length === 0
+                                ? 'loading'
+                                : 'ready'
+                        }
+                        features={dataList}
                         environments={environments}
-                        loading={loading}
+                        loading={loading && dataList.length === 0}
```

Comment:
What if the project does not have any features?

Reply:
No feature toggles available screen as before